src/main/java/com/dougnoel/sentinel/steps/BaseSteps.java

```diff
@@ -46,7 +52,23 @@ public static void before(Scenario scenario) {
     @When("^I click (?:the|a|an) (.*?)$")
     public static void click(String elementName) {
         getElement(elementName).click();
-    }
+    }    
+
+    /**
+     * Moves the mouse to the middle of the element
+     * <p>
+     * <b>Gherkin Examples:</b>
+     * <ul>
+     * <li>When I mouse over element</li>
+     * </ul> 
+     * @param elementName String the target element to move to
+     */
+    
+     @When("^I mouse over (.*?)$") 
+    public void mouseoveronTarget(String elementName) {    
```

Comment:
Remove this method from this file and instead update the TextVerificationSteps and add the ability to check the tooltip text of an item as part of the text equals/contains step.

Reply:
I updated the method to get the tool tip text from the page element